docs/source/rabbitmq-tutorial/examples/7-publisher-confirms/publish_asynchronously.py

```diff
@@ -0,0 +1,58 @@
+import asyncio
+
+from aio_pika import Message, connect
+from aiormq.exceptions import DeliveryError
+from pamqp.commands import Basic
+
+
+def get_messages_to_publish():
+    for i in range(10000):
+        yield f"Hello World {i}!".encode()
+
+
+async def publish_and_handle_confirm(exchange, queue_name, message_body):
+    try:
+        confirmation = await exchange.publish(
+            Message(message_body),
+            routing_key=queue_name,
+            timeout=5.0,
+        )
+    except DeliveryError as e:
+        print(f"Delivery of {message_body!r} failed with exception: {e}")
+    except TimeoutError:
+        print(f"Timeout occured for {message_body!r}")
+    else:
+        if not isinstance(confirmation, Basic.Ack):
+            print(f"Message {message_body!r} was not acknowledged by broker!")
+
+
+async def main() -> None:
+    # Perform connection
+    connection = await connect("amqp://guest:guest@localhost/")
+
+    async with connection:
+        # Creating a channel
+        channel = await connection.channel()
+
+        # Declaring queue
+        queue = await channel.declare_queue("hello")
+
+        async with asyncio.TaskGroup() as tg:
```

Comment:
If we use this here, you should explicitly specify that these examples are written for python 3.11, and will not run in versions below it.